macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -154,3 +162,6 @@ class ColorRectView: NSView {
 }
 
 private let colorRowSpacing: CGFloat = 10.0
+
+// Default to using the new ColorProvider protocol for fetching the Fluent Primary Colors
+private var useColorProvider: Bool = true
```

Comment:
Tiny bit of brevity without loss of clarity - can use type inference for this one: `private var useColorProvider = true`